ezGroceriesShoppingList/src/main/java/com/ezgroceries/shoppinglist/service/CocktailServiceImpl.java

```diff
@@ -0,0 +1,59 @@
+package com.ezgroceries.shoppinglist.service;
+
+import com.ezgroceries.shoppinglist.entity.Cocktail;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class CocktailServiceImpl implements CocktailService {
+
+    @Override
+    public List<Cocktail> getCocktails(String search) {
+        Cocktail firstCocktail = new Cocktail(
+                "23b3d85a-3928-41c0-a533-6538a71e17c4",
+                "Margerita",
+                "Cocktail glass",
+                "Rub the rim of the glass with the lime slice to make the salt stick to it. Take care to moisten..",
+                "https://www.thecocktaildb.com/images/media/drink/wpxpvu1439905379.jpg",
+                new ArrayList() {
+                    {
+                        add("Tequila");
+                        add("Blue Curacao");
+                        add("Lime juice");
+                        add("Salt");
+                    }});
+        Cocktail secondCocktail = new Cocktail(
+                "d615ec78-fe93-467b-8d26-5d26d8eab073",
+                "Blue Margerita",
+                "Cocktail glass",
+                "Rub rim of cocktail glass with lime juice. Dip rim in coarse salt..",
+                "https://www.thecocktaildb.com/images/media/drink/qtvvyq1439905913.jpg",
+                new ArrayList() {
```

Comment:
One easier way to add objects to a list is the Method List.of(obj1, obj2, obj3). But that's fine also